src/css/Books.css

```diff
@@ -0,0 +1,183 @@
+.panel-bg {
+  display: flex;
+  background-color: #fff;
+  justify-content: space-between;
+  padding-left: 5%;
+  margin-right: 5%;
+  align-items: center;
+  height: 25%;
+  width: 85vw;
+  border-radius: 10px;
+  margin-bottom: 2%;
+}
+
```

Comment:
Great job with your design so far, however, there are just few adjustments to be made 😃. Kindly note that we want the styling to be **exactly** the same as can be seen in the Zeplin template. Kindly ensure to improve upon the following:
- [ ] Typeface of book titles should be _RobotoSlab-Bold_, and also implement the corresponding font-size and color as specified in the Zeplin. Hint: Simply click on the specific text to display its properties on a right sidebar
- [ ] Current chapter text should have typeface as _RobotoSlab-Light_, size should be _0.813rem_ and color should be _#121212_ as can be seen in the template on Zeplin.
- [ ] Properties of all other fonts should also correspond as specified in the Zeplin template (**Type, size and color**) 
- [ ] The size of the percentage statistics found in the middle should match the Zeplin template.
- [ ] Each book entry should have thine grey rounded border and appropriate spacing separating it from previous entry.
- [ ] The form inputs should have spacing corresponding to the one found in the Zeplin
- [ ] Buttons should be of the appropriate sizes and color matching the Zeplin template.


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/192125358-5a71ad7c-da87-4836-b582-f98afafef8dc.png)

**The Zeplin design:**
![image](https://user-images.githubusercontent.com/12745474/191832604-be0376a8-3b3c-4d5e-aefa-446c69e1cdbc.png)